src/main/java/com/course/fleet/management/API/controller/UsersController.java

```diff
@@ -0,0 +1,39 @@
+package com.course.fleet.management.API.controller;
+
+import com.course.fleet.management.API.entity.User;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import net.datafaker.Faker;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@RestController
+@RequestMapping("fleet-management-API/users")
+public class UsersController {
+
+    private static final List<User> users;
+
+    static {
+        Faker faker = new Faker();
+        users = IntStream.range(0,10)
+                .mapToObj(i -> {
+                    String firstName = faker.name().firstName();
+                    String lastName = faker.name().lastName();
+                    return new User(
+                            (long) i,
+                            firstName + " " + lastName,
+                            firstName + "." + lastName + "@mail.com"
+                    );
+                })
+                .collect(Collectors.toList());
+
+    }
+    @GetMapping
+    public ResponseEntity <List<User>> getUsers(){
```

Comment:
Hay que cuidar estos espacios por si el proyecto en el que se está trabajando hace uso de un sonar como por ejemplo sonarcloud.

```suggestion
    public ResponseEntity<List<User>> getUsers() {
```